crates/volta-core/src/version/serial.rs

```diff
@@ -1,45 +1,45 @@
-use semver::{ReqParseError, VersionReq};
+use semver::{Compat, ReqParseError, VersionReq};
```

Comment:
I'll add a comment about that.

We need `Compat` and `parse_compat` because the `semver` crate defaults to parsing in a cargo-compatible way. This is normally fine, except for 2 cases (that I know about):

* `"1.2.3"` parses as `^1.2.3` for cargo, but `=1.2.3` for Node
* `>1.2.3 <2.0.0` serializes to `">1.2.3, <2.0.0"` for cargo (with the comma), but `">1.2.3 <2.0.0"` for Node (no comma, because Node parses commas differently)

This specifically affects this PR because we are parsing the version requirements from the command line, then serializing them to pass to `npm view`, so they need to be handled in a Node-compatible way (or we get the wrong version info returned).